examples/vpnaas/endpoint_group/main.tf

```diff
@@ -0,0 +1,8 @@
+resource "vkcs_vpnaas_endpoint_group" "allowed_hosts" {
+  name = "allowed-hosts-tf-example"
+  type = "cidr"
```

Comment:
we need an example for another type of endpoint group as well

Reply:
added it in subnet subfolder